cpp/README.md

```diff
@@ -250,6 +250,23 @@ Logging IWYU to /tmp/arrow-cpp-iwyu.gT7XXV
 ...
 ```
 
+### Linting
+
+We require that you follow a certain coding style in the C++ code base.
+You can check your code abides by that coding style by running:
+
+    make lint
+
+You can also fix any formatting errors automatically:
+
+    make format
+
+These commands require `clang-format-4.0` (and not any other version).
+You may found the required packages for Debian/Ubuntu on https://apt.llvm.org/.
```

Comment:
Thanks for adding this! Maybe also add http://releases.llvm.org/download.html which has binaries for Windows, macOS, and other Linuxes

Reply:
Done.